src/controller/order-controller.ts

```diff
@@ -127,3 +162,70 @@ export async function captureOrderController(fastify: FastifyInstance) {
     },
   });
 }
+
+// Patch order
+export type ShippingOption = {
+  id: string;
+  label: string;
+  type: string;
```

Comment:
I wonder if it's worth using a literal union for `type` here, maybe something like:

 `type: "SHIPPING" | "PICKUP" | (whatever the other options are)`
 
 Seems like a good practice to teach, and might make it slightly more clear to readers what this field is meant to represent.

Reply:
Good suggestion @dtjones404! We may be able to leverage paypal-js for this type.

Reply:
Would this type work? https://github.com/paypal/paypal-js/blob/9e3472a26d28f971fbe2d90bad7b57c969f4a8b5/types/apis/shipping.d.ts#L10

Reply:
Looks great!